docs/data/rpc/admin-guide.mdx

```diff
@@ -287,13 +287,24 @@ HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_003/{
 
 3. Run the image, making sure to [mount a volume](https://docs.docker.com/storage/volumes/) to your container where the above configuration is stored. An example for Testnet:
 
+If you are running locally on a macOS for instance, you could create a volume in your home directory: `~/test-rpc-config`
+
+Then you would add the following config files to that local directory:
+
+- soroban-rpc-config.toml
+- stellar-captive-core.cfg
+
+Then you would mount that volume using the following parameter: `-v /Users/<USERNAME>/test-rpc-config:/opt/stellar`.
+
+Your running container would mount that volume at the path `/opt/stellar`
+
+In that example, you could run the `stellar/stellar-rpc` container with the following command:
+
 ```bash
 docker run -p 8001:8001 -p 8000:8000 \
--v <STELLAR_CORE_CONFIG_FOLDER>:/config stellar/stellar-rpc \
---captive-core-config-path="/config/<STELLAR_CORE_CONFIG_PATH>" \
+-v /Users/<USERNAME>/test-rpc-config:/opt/stellar stellar/stellar-rpc \
+--captive-core-config-path=/opt/stellar/stellar-captive-core.cfg \
 --captive-core-storage-path="/var/lib/stellar/captive-core" \
---captive-core-use-db=true \
---stellar-core-binary-path="/usr/bin/stellar-core" \
```

Comment:
is the `--stellar-core-binary-path` argument not needed anymore? just wanted to be sure

Reply:
There's a common-sense default: 

https://github.com/stellar/stellar-rpc/blob/main/cmd/stellar-rpc/internal/config/options.go#L147
```
		{
			Name:         "stellar-core-binary-path",
			Usage:        "path to stellar core binary",
			ConfigKey:    &cfg.StellarCoreBinaryPath,
			DefaultValue: defaultStellarCoreBinaryPath,
			Validate:     required,
		},
```

The command is pretty verbose so I shaved it off but let me know what you think?

Reply:
yeah, i don't think there's any harm either way. you'd probly have to know pretty well what/why you were doing, if you wanted to specify a non-default value. i think it's fine to leave it off